go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -4906,3 +4906,100 @@ END`,
 		},
 	},
 }
+
+var DoltForeignKeyStatus = []queries.ScriptTest{
+	{
+		Name: "Show unresolved foreign key status",
```

Comment:
You should also have tests that check for the different failure conditions in your implementation, like calling the function with inputs of the wrong type, or with unresolvable references, etc.